worker/src/main.rs

```diff
@@ -160,18 +170,17 @@ fn worker(node_url: &str, w_ip: &str, w_port: &str, mu_ra_port: &str) {
 		},
 	};
 
+	println!("*** call enclave init()");
```

Comment:
cosmetic: but we have an `init_enclave()` function and an `init()` ecall to the enclave. Would suggest better naming to see the difference between the two

Reply:
agreed. but let's fix that later